### PR TITLE
test PR

### PR DESCRIPTION
**Pull request by opanitch**
this is a test pull request to show the review workflow

### CODE REVIEW

**Review by hdmark — COMMENTED** [review]
wooohooo

File: app/index.ejs [hunk 1]
```diff
@@ -9,6 +9,7 @@
         <link rel="stylesheet" href="<%= htmlWebpackPlugin.files.chunks.main.css %>">
     </head>
     <body>
+        <h1>Hello World</h1>
```
Comment (opanitch) [inline app/index.ejs:12]: This is a stupid message - let's change this title

File: app/index.ejs [hunk 2]
```diff
@@ -9,6 +9,7 @@
...
         <div id="app"></div>
         <script>
```
Comment (opanitch) [inline app/index.ejs:14]: secnd comment

**Review by hdmark — CHANGES_REQUESTED** [review]
this is no good

**Review by hdmark — APPROVED** [review]
looks good

**Pull request merged**